examples/CAgent/c_agent/agent.py

```diff
@@ -69,7 +69,8 @@
 __version__ = 1.0
```

Comment:
__version__ should be a string.
https://www.python.org/dev/peps/pep-0396/#specification